packages/sit-onyx/tsconfig.playwright.json

```diff
@@ -4,6 +4,7 @@
   "exclude": [],
   "compilerOptions": {
     "composite": true,
+    "lib": ["es2021"],
```

Comment:
Why is this needed?

Reply:
`replaceAll` is not available in `es2016`